Detectors/TPC/simulation/src/SAMPAProcessing.cxx

```diff
@@ -65,17 +68,18 @@ bool SAMPAProcessing::importSaturationCurve(std::string file)
   return true;
 }
 
-void SAMPAProcessing::getShapedSignal(float ADCsignal, float driftTime, std::array<float, mNShapedPoints>& signalArray)
+void SAMPAProcessing::getShapedSignal(float ADCsignal, float driftTime, std::array<float, 8> &signalArray)
```

Comment:
I changed the array to a std::vector to maintain the possibility to modify the size - in the end we should optimize this anyhow